src/auth/screens/auth-profile.screen.js

```diff
@@ -121,9 +121,9 @@ class AuthProfile extends Component {
           renderContent={() =>
             <UserProfile
               type="user"
-              initialUser={hasInitialUser ? user : {}}
-              user={hasInitialUser ? user : {}}
-              starCount={hasInitialUser ? starCount : ''}
+              initialUser={hasInitialUser && !!starCount ? user : {}}
```

Comment:
We have a lot of double negations in out app and I think we may be better to start leveraging `lodash` properly. Let's discuss this after our release however.